### ISSUE TITLE
Check colours of example graphs to ensure accessibility

### ISSUE THREAD

**Issue by DragaDoncila** [live-image-tracking-tools/traccuracy#271]
Our example graphs have dark blue against black contrast, and orange/purple which may not be sufficiently high-contrast for accessibility reasons. We should review the colours and pick a palette that is maximally accessible (including being colorblind friendly).
